cpp/src/bitmask/null_mask.cu

```diff
@@ -135,7 +135,8 @@ void set_null_mask(bitmask_type *bitmask,
 {
   CUDF_FUNC_RANGE();
   CUDF_EXPECTS(begin_bit >= 0, "Invalid range.");
-  CUDF_EXPECTS(begin_bit < end_bit, "Invalid bit range.");
+  CUDF_EXPECTS(begin_bit <= end_bit, "Invalid bit range.");
```

Comment:
I feel like these checks should be either merged or we should have more detailed error messages so that the separate checks are useful.

Reply:
Looks like these assertions have a similar issue: 

https://github.com/rapidsai/cudf/blob/f30be676198a3085914dbeff5b911715a41aeebd/cpp/src/bitmask/null_mask.cu#L387-L388
https://github.com/rapidsai/cudf/blob/f30be676198a3085914dbeff5b911715a41aeebd/cpp/src/bitmask/null_mask.cu#L478-L479

I didn't bother to see if there are more, but I feel improving such checks deserves a separate PR.